nginx-configs/production/image_server_to_iiif.js

```diff
@@ -0,0 +1,55 @@
+var typeToDimensionMapping = {
+  "v": "2560",
+  "q": "1600",
+  "w": "760",
+  "r": "300",
+  "t": "150",
+  "f": "140",
+  "b": "100"
+}
+​
+function mapImageServerToIIIF(request, response) {
+​
+  // turn query string to hash
+  var requestVars = request.variables['query_string'];
+  var arrayOfKeyValues;
+  request.log("request vars: " + requestVars);
+  if (requestVars) {
+  	arrayOfKeyValues = requestVars.split('&');
+  }
+  var paramsHash = {};
+​
+​
+  // Turns ['a=1', 'b=2', 'c=3'] and stores it into paramsHash as {"a": "1", "b": "2", "c": "3"}
+  for (var i = 0; i < arrayOfKeyValues.length; i++) {
+    paramsHash[arrayOfKeyValues[i].split('=')[0]] = arrayOfKeyValues[i].split('=')[1];
+  }
+​
+  // Dimension Stuff
+  var imageType = paramsHash['t'].toLowerCase();
+  var dimension = typeToDimensionMapping[imageType];
+  var urlSegment;
+  var crop;
+​
+  if (imageType == 'u' || imageType == 'j' || imageType == 's') {
+    urlSegment = "full";
+  } else {
+    urlSegment = "!" + dimension + "," + dimension;
+  }
+  var identitifier = paramsHash['id']
+​
+  if (paramsHash['t'] == 'b') {
+    crop = "square"
+  } else {
+    crop = "full"
+  }
+​
+  var imageUrl = "http://172.16.1.205:8182/iiif/2/"+ identitifier + "/" + crop + "/" + urlSegment +"/0/default.jpg"
```

Comment:
I thought production was *.195 (and *.194?) and qa *.193?

Reply:
You might be right. I was going off information from Brett, but I might have misunderstood. Will confirm with him the right ips tomorrow.

Reply:
Using local for both now. You are correct. Done.